README.md

```diff
@@ -3,59 +3,44 @@
 ## Mingsplosion
 
 #### Decisions
-- Added **form Empire of China** decision
-
-#### Government reforms
-- Added **Tier 1 government reform** which is only available to countries with **Confucian religion** and that are both **<145 dev** and don't have **Chinese culture group primary culture**, grants **+5% morale of armies** and **+10% goods produced**.  A version of this reform also exists for **republics**, it also adds **-40 absolutism**
-- Added **Tier 1 government reform** which is only available to countries with **Chinese culture group primary culture**, **Confucian religion** and **>144 dev**, grants **+5% morale of armies**, **-10% aggressive expansion**, **+10% goods produced** and **+100 governing capacity**. A version of this reform also exists for **republics**, it also adds **-40 absolutism**
-- Added **Tier 1 government reform** which is only available to countries with **Empire of China tag** with **Confucian religion**, grants **+10% morale of armies**, **+0.2% yearly army professionalism**, **+20% goods produced** and **+300 governing capacity**. Only available to monarchies
-- Added **Tier 2 government reform** which is only available to countries with **Empire of China tag**, grants **+20% manpower modifier** and **+20% land force limit**. Only available to monarchies
-- Added **Tier 4 government reform** which is only available to countries with **Empire of China tag** with **Confucian religion**, grants **+2 monarch admin power** and **-1 diplomatic relations**. Only available to monarchies
+- Added **form Empire of China** decision which forms **Ming** and restores the **Mandate of Heaven** if it has been disbanded
 
 #### History
 - **Ming** broken up into multiple nations, many with **unique national ideas**
-
-#### Mandate of Heaven
-- Completely removed the **Mandate of Heaven** mechanic
+- All Chinese nations are now **daimyo type vassals** to the **Chinese emperor**, allowing them to fight internally
 
 #### Triggered Modifiers
-- Added modifier, all **Theocracies** with **Confucian** religion gain **+5% morale of armies** and **+10% goods produced**
+- The **Chinese emperor** always get **+0.1 mandate growth**
 
-#### Wargoals
-- Changed **Unify China** CB. Now only available **before 1550** to countries with **Chinese culture group primary culture** or **Manchu tag** or **Qing tag**. Applies to all provinces within **China super region**. Grants **75% aggressive expansion**, **150% prestige** and **100% peace cost**
+## Alliance mod
+
+- Custom system which keeps track of player-player alliances
```

Comment:
This change is not added in this MR, gurkan will add in separate MR. Preemptively  added it to changelog.